packages/faency/src/components/Switch.story.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { storiesOf } from '@storybook/react'
+import { Box } from './Box'
+import { Button } from './Button'
+import { Text } from './Text'
+import { Switch } from './Switch'
+
+function SwitchStory() {
+  const [checked, setChecked] = useState(false)
+
+  return (
+    <>
+      <Box mb="4">
+        <Text as="p">Uncontrolled</Text>
+        <Switch value="row" defaultChecked mr={1} />
+      </Box>
+
+      <Box mb="4">
+        <Text as="p">Uncontrolled and without children</Text>
+        <Switch value="true" defaultChecked />
+      </Box>
```

Comment:
It seems to have no difference between the first and the second case. What is the difference? And what the `value="row"` stand for?

Reply:
@paulocfjunior Fixed the unnecessary example. "row" is the value of the input. It's working the exact same way as an checkbox native element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Reply:
Got it, thanks for the link, that was very clarifying